kernel/src/common/utils.rs

```diff
@@ -210,6 +210,6 @@ macro_rules! create_capability {
 #[macro_export]
 macro_rules! count_expressions {
     () => (0usize);
-    ($head:expr) => (1usize);
-    ($head:expr, $($tail:expr),*) => (1usize + count_expressions!($($tail),*));
+    ($head:expr $(,)?) => (1usize);
+    ($head:expr, $($tail:expr),* $(,)?) => (1usize + count_expressions!($($tail),*));
```

Comment:
I don't speak Rust macro that well, but it looks like this might also be a case where the trailing comma is currently required (as part of the `$($tailExpr),*` pattern?) and the last `$(,)?` isn't actually ever going to match?

Reply:
https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=4c2e7eb72007eda40ad5b974a54d0dc8

I have to admit, it's a quite confusing syntax. :laughing: However it's still required, since the repeated match (is there a more precise name for this?) only specifies the delimiter _between_ the items, so any trailing comma needs to be matched separately.